### ISSUE TITLE
Can config file be used with targetRepositoryName?

### ISSUE THREAD

**Issue by bdukes** [tinglesoftware/dependabot-azure-devops#357]
I setup a new repo and pipeline to centralize using Dependabot, and created a task like this:
```yaml
task: dependabot@1
inputs:
  useConfigFile: true
  gitHubConnection: GitHub - read for dependabot
  targetRepositoryName: Engage1
```

However, the build failed with the error `Configuration file not found at /.azuredevops/dependabot.yml or /.github/dependabot.yml`

The repository in question did have a file at `/.azuredevops/dependabot.yml` which I had used successfully to work with Dependabot before moving to the centralized repo & pipeline. So it seems that the task isn't able to get the config from the remote repository. Is it expected that this would work? I don't mind having the config in the centralized pipeline's YAML, but I'm concerned about the warning I now get:

>        Using explicit inputs instead of a configuration file will be deprecated in the next minor release.
>
>        Migrate to using a config file at .azuredevops/dependabot.yml or .github/dependabot.yml.
>
>        See https://github.com/tinglesoftware/dependabot-azure-devops/tree/main/src/extension#usage for more information.

**Comment by mburumaxwell** [comment]
@bdukes indeed this will not work. I only found out this in the morning because reading of the configuration file requires access to the repository.
Closing this in favour of #328

**Comment by mburumaxwell** [comment]
Fixed in #444
